common/lib/xmodule/xmodule/video_module/video_utils.py

```diff
@@ -57,6 +57,12 @@ def rewrite_video_url(cdn_base_url, original_video_url):
         return None
 
     parsed = urlparse(original_video_url)
+
+    # Only re-write URLs if the original netloc is in the whitelist, otherwise
+    # no guarantees can be made that the re-write target can service the request.
+    if parsed.netloc not in whitelist:
```

Comment:
Wondering if you need a try/catch in here in case `whitelist` ends up being derived from a badly-written setting?